src/main/java/org/fcrepo/spec/testsuite/AbstractTest.java

```diff
@@ -549,8 +550,13 @@ protected void confirmPresenceOfLinkValue(final Link linkValue, final Response r
 
     protected void confirmPresenceOfConstrainedByLink(final Response response) {
         final String contrainedByUri = "http://www.w3.org/ns/ldp#constrainedBy";
-        assertTrue("Response does not contain link of rel type = " + contrainedByUri,
-                   getLinksOfRelType(response, contrainedByUri).count() > 0);
+        final Optional<URI> link = getLinksOfRelTypeAsUris(response, contrainedByUri).findAny();
+        if (!link.isPresent()) {
+            fail("Response does not contain a link of rel type = " + contrainedByUri);
```

Comment:
might as well fix the typo while we're here, `constrainedByUri`

Reply:
I'm also not clear why we now need to verify that the constrainedby link is resolvable? Not that its a bad thing to check

Reply:
@bbpennel : Re resolution:  So is it possible that a constrainedBy URI would not be resolvable? In other words, of what use would an unresolvable be?